README.md

```diff
@@ -97,6 +97,21 @@ programs, even if they have type errors:
 [statically typed parts]: http://mypy.readthedocs.io/en/latest/basics.html#function-signatures
 
 
+Integrations
```

Comment:
Maybe should also mention @ambv's https://github.com/ambv/flake8-mypy, which is a flake8 plugin and therefore usable in any editor that has a flake8 integration.